src/PortalWithState.js

```diff
@@ -73,7 +73,7 @@ class PortalWithState extends React.Component {
     if (!this.state.active) {
       return;
     }
-    const root = findDOMNode(this.portalNode);
+    const root = findDOMNode(this.portalNode.defaultNode);
```

Comment:
@tajo not sure about this, but this was how I got it to work in react 15.6.2 (also still works in react16). Before, findDOMNode was returning `null` if we didn't pass it the defaultNode

Reply:
actually with this, we don't need findDOMNode, we can just access it directly with `this.portalNode.defaultNode`